lms/djangoapps/class_dashboard/dashboard_data.py

```diff
@@ -21,6 +23,23 @@
     'lti',
 ]
 
+# exclude these in Metrics
+non_student_roles = ['instructor', 'staff']
+
+def get_non_student_list(course_key):
+    # find all user_ids with instructor or staff roles in student_courseaccessrole table
+    non_student_idlist=[]
```

Comment:
style note:  want whitespaces on each side for assignments.

Python's style guide is https://www.python.org/dev/peps/pep-0008/

doing `paver run_quality` will run pylint and pep8 on the diff you're introducing.  Try not to introduce any violations.